src/Umbraco.Web.UI.Client/src/views/packages/views/repo.html

```diff
@@ -187,6 +187,7 @@ <h4><strong><localize key="general_searchNoResult">Sorry, we can not find what y
                         ng-if="vm.lightbox.show"
                         items="vm.lightbox.items"
                         active-item-index="vm.lightbox.activeIndex"
+                        ng-attr-umb-focus-lock="true"
```

Comment:
Does it need the `ng-attr-` part here?

Reply:
@bjarnef I acctuly don´t know. I put more weight on what there has done in the past. See https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/views/components/editor/umb-editors.html line  5 

First I did this PR based on above files:
https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Cms.StaticAssets/umbraco/UmbracoBackOffice/Default.cshtml line 91

And now this PR, so it was more an alignment, maybe is something to do white the ng-attr tell to run expresstion and just umb-focus-lock would bee a binding to the Angular directive scope

Reply:
Okay, I only think it makes a difference to ensure to binding is ready, e.g. `style` vs `ng-style`. I recall originalen IE had issues when using expression directly in `style` attribute.

Reply:
But this should work as well: https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/views/components/overlays/umb-overlay.html#L5 (default is true).

Reply:
I did a small test. It dons´t matter if you type in `ng-attr-umb-focus-trap` or just `umb-focus-trap`. Both work and fix the modal trap issue. So it is more what will be more likeable to look at?

Reply:
I guess you mean `umb-focus-lock` 😉

Reply:
yea - need a Redbull